ddl/backfilling.go

```diff
@@ -198,9 +211,13 @@ type backfillCtx struct {
 	batchCnt   int
 }
 
-func newBackfillCtx(ctx *ddlCtx, sessCtx sessionctx.Context, reorgTp model.ReorgType,
+func newBackfillCtx(ctx *ddlCtx, id int, sessCtx sessionctx.Context, reorgTp model.ReorgType,
 	schemaName string, tbl table.Table) *backfillCtx {
+	if id == 0 {
+		id = int(backfillContextID.Add(1))
+	}
 	return &backfillCtx{
+		id:         id,
```

Comment:
Why add `id` here? And it looks different from the backfillJob id. Not get point here.

Reply:
We use it in `func (w *backfillWorker) String() string `.

Reply:
ok, seems `id` only for log.